File: py/1893.py
def main() -> None:
    a, b = [int(i) for i in input().split()]
    if 0 <= b <= 2:
        print('nova')
    elif 97 <= b <= 100:
        print('cheia')
    elif a < b:
        print('crescente')
    else:
        print('minguante')


if __name__ == '__main__':
    main()

File: cpp/1893.cpp
#include <iostream>

using namespace std;

int main()
{
    int a, b;
    cin >> a >> b;
    if (0 <= b && b <= 2) {
        cout << "nova\n";
    }
    else if (97 <= b && b <= 100) {
        cout << "cheia\n";
    }
    else if (a < b) {
        cout << "crescente\n";
    }
    else {
        cout << "minguante\n";
    }
    return 0;
}